android/build.gradle

```diff
@@ -14,35 +14,44 @@ useExpoPublishing()
 // Most of the time, you may like to manage the Android SDK versions yourself.
 def useManagedAndroidSdkVersions = false
 if (useManagedAndroidSdkVersions) {
-  useDefaultAndroidSdkVersions()
+    useDefaultAndroidSdkVersions()
 } else {
```

Comment:
this will always go to else? No need for the if then?

Reply:
This is generated code from expo, not too sure about it. Just reformatted it.